boxtree/cost.py

```diff
@@ -34,14 +34,11 @@
 multipole expansion to a single target.
 
 :class:`AbstractFMMCostModel` uses :class:`FMMTranslationCostModel` and calibration
-parameter to compute the total cost of each step of FMM in each box. There are two
-implementations of the interface :class:`AbstractFMMCostModel`, namely
-:class:`CLFMMCostModel` using OpenCL and :class:`PythonFMMCostModel` using pure
-Python. The calibration parameter can be estimated using
-:meth:`AbstractFMMCostModel.estimate_calibration_params`.
+parameter to compute the total cost of each step of FMM in each box. There is an
+:class:`AbstractFMMCostModel`, implemented by :class:`FMMCostModel`.
```

Comment:
Is `FMMCostModel` here a typo or are you suggesting we should rename every `CLFMMCostModel` to `FMMCostModel`?

Reply:
I was tempted to do that, but I backed off. What do you think?

Reply:
Matt has given a similar suggestions before: https://gitlab.tiker.net/inducer/boxtree/merge_requests/72#note_30988

Now I think this suggestion makes sense, especially after we explicitly say the user should not use the Python implementation in https://github.com/gaohao95/boxtree/pull/1/files#diff-c32e45365ecffd35bec421e3e273baeaR1224.

Reply:
I will make the changes after this get merged.

Reply:
Sounds good. Anything else you'd like changed or would like to discuss in this (meta-)PR?

Reply:
Let me take a more detailed look tomorrow and change the name to `FMMCostModel`. I will let you know once everything is ready.